README.md

```diff
@@ -1188,6 +1191,39 @@ All options from BDD tests do also apply here.
 
 **NOTE:** Performance tests should be run without concurrency.
 
+#### List of test tags
+
+| Tag name           | Tag meaning                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------- |
+| @smoke             | Smoke tests verifying basic functionality                                          |
+| @performance       | Performance tests                                                                  |
+| @olm               | OLM integration tests                                                              |
+| @travelagency      | Travel agency tests                                                                |
+|                    |                                                                                    |
+| @disabled          | Disabled tests, usually with comment describing reasons                            |
+| @cli               | Tests to be executed only using Kogito CLI                                         |
+|                    |                                                                                    |
+| @springboot        | SpringBoot tests                                                                   |
+| @quarkus           | Quarkus tests                                                                      |
+|                    |                                                                                    |
+| @dataindex         | Tests including DataIndex                                                          |
+| @jobsservice       | Tests including Jobs service                                                       |
+| @managementconsole | Tests including Management console                                                 |
+| @infra             | Tests checking KogitoInfra functionality                                           |
+|                    |                                                                                    |
+| @binary            | Tests using Kogito applications built locally and uploaded to OCP as a binary file |
+| @native            | Tests using native build                                                           |
+| @persistence       | Tests verifying persistence capabilities                                           |
+| @events            | Tests verifying eventing capabilities                                              |
```

Comment:
Just thinking about `@events`, there are of 2 types or ?

- Send process instances to Kafka (and then retrieved by Data Index)
- Communication between services (like travelagency) via Kafka

We have not annotated the travelagency with any for now but it is using events.

Should we differentiate the two different ? Would we want that ?

Reply:
For me events are "our" events, meaning these implemented directly by Kogito. If events are implemented as a 3rd party feature for communication between KogitoApps by a developer, I wouldn't use it for these scenario. (Which is what Travel Agency implements if I understand)

Reply:
well, events are set in the BPMN process.
we still need to be sure that the communication is working good or ?

Reply:
hmm, good question.
From my point of view it depends on the purpose of Kafka configuration made by Kogito operator. Is this configuration meant to be used by eventing capability of processes? Or is it meant just for communication with Data index and possibility of reusing it for process messaging is just a side effect?

Considering that user can use any messaging provider for Kogito I would guess the latter is more likely. So in such case I would mark tests with `@events` just for Data index integration.
Anyway the eventing in Travel agency tests is an implementation detail for us (as the purpose of our scenario is to check that Travel agency example works without any specific knowledge of implementation) so I wouldn't annotate it with `@events`.

Reply:
oki. good for me then if only dataindex is instantiated